ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane-savage.ts

```diff
@@ -951,36 +951,42 @@ const triggerSet: TriggerSet<Data> = {
         goLeft3Right2: {
           en: 'Go 3 Left 2 Right',
           de: 'Gehe Links 3, Rechts 2',
+          fr: '3 à gauche, 2 à droite',
```

Comment:
These edits should be made in another_sildihn_subterrane-savage.ts and util/sync_files.ts ran.

Reply:
Yes, I agree.

Sorry for any confusion.  I added https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane-savage.ts#L1-L2 to try to explain this, but you're not the first person who has edited this file directly instead of editing the non-savage one and running sync_files.